### PR TITLE
Fix bugs in the GenevaExporter PrepopulatedField handling

### PR DESCRIPTION
**Pull request by reyang**
Addressing https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/514#discussion_r929575424 and https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/514#discussion_r929576498.

FYI @xyq175com

### CODE REVIEW

File: src/OpenTelemetry.Exporter.Geneva/GenevaExporterOptions.cs [hunk 1]
```diff
@@ -105,7 +105,12 @@ public class GenevaExporterOptions
                     case string vs:
                         break;
                     default:
-                        throw new ArgumentException($"Type: {entry.Value.GetType()} is not supported. Only bool, byte, sbyte, short, ushort, int, uint, long, ulong, float, double, and string are the supported types for PrepopulatedFields values.");
+                        if (entry.Value == null)
+                        {
+                            throw new ArgumentException($"Null is not allowed (key = `{entry.Key}`).");
```
Comment (utpilla) [inline src/OpenTelemetry.Exporter.Geneva/GenevaExporterOptions.cs:110]: Do we want to do something similar to what we have for `TableNameMappings` to stay consistent? https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Exporter.Geneva/GenevaExporterOptions.cs#L49

**Review by utpilla — APPROVED** [review]

File: src/OpenTelemetry.Exporter.Geneva/GenevaExporterOptions.cs (hunk 1 above)
Comment (reyang) [reply src/OpenTelemetry.Exporter.Geneva/GenevaExporterOptions.cs]: Good point, I've made the updates.

**Comment by xyq175com** [comment]
@reyang  
Really sorry for the issue, i thought that was the minor suggestions and not need to update..

**Comment by reyang** [comment]
> @reyang Really sorry for the issue, i thought that was the minor suggestions and no need to update..

No worries 😺

**Pull request merged**